erikiado/jp2_online
# Task delete transacciones

#### What's this PR do?
It allows a capturista to delete a transaccion from a family.

#### Where should the reviewer start?
The reviewer should check the view for getting the delete modal
The reviewer should then check the view for actually deleting the transaccion
Then the reviewer should check the form for deleting

#### How should this be manually tested?
Go to the transaccion page of any family
Click the delete button on any transaccion
Click the confirm button
Check that the transaccion doesn't appear and that the economic status table has changed

#### Any background context you want to provide?
This pull request was opened before the tests are completed in order to allow @csssaz and @fernandolobato to check the code before they travel tomorrow


This template was adapted ~stolen~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)
